Mathlib/Algebra/Algebra/Operations.lean

```diff
@@ -163,6 +163,9 @@ theorem mul_le_mul_left (h : M ≤ N) : M * P ≤ N * P :=
 theorem mul_le_mul_right (h : N ≤ P) : M * N ≤ M * P :=
   AddSubmonoid.mul_le_mul_right h
 
+theorem mul_comm_of_commute (h : ∀ m ∈ M, ∀ n ∈ N, Commute m n) : M * N = N * M :=
+  toAddSubmonoid_injective <| AddSubmonoid.mul_comm_of_commute h
```

Comment:
This won't work without the dependency PR which makes `(M * N).toAddSubmonoid` defeq to `M.toAddSubmonoid * N.toAddSubmonoid`.